vm/interpreter/src/default_runtime.rs

```diff
@@ -126,6 +131,11 @@ where
             value_received: message.value().clone(),
         };
 
+        let default_preignition =
+            setup_preignition_genesis_actors_testnet(&gas_block_store).unwrap();
+        let default_postignition =
+            setup_postignition_genesis_actors_testnet(&gas_block_store).unwrap();
```

Comment:
I don't see why you would possibly need the unwrap? just downcast to an `ActorError`